Makefile

```diff
@@ -1,14 +1,33 @@
+# Auto-generated by Cimas: Do not edit it manually!
+# See https://github.com/metanorma/cimas
+#!make
+ifeq ($(OS),Windows_NT)
+SHELL := cmd
+else
+SHELL := /bin/bash
+endif
+
 SRC := $(wildcard views/*.lutaml)
-PNG := $(patsubst views/%.lutaml,images/%.png,$(SRC))
 
-SHELL := /bin/bash
+ifeq ($(SRC),)
+SRC := $(patsubst models/%.wsd,views/%.lutaml,$(SRC))
+endif
+
+PNG := $(patsubst views/%.lutaml,images/%.png,$(SRC))
 
 all: $(PNG)
 
 images/%.png: views/%.lutaml
 	lutaml -t png -o $@ $<
 
+views/%.lutaml: models/%.wsd
```

Comment:
There is no `.wsd` file in this repository.

Reply:
Right, but I will add this Makefile to cimas to so we will use a single Makefile for all metanorma-model-* repositories

Reply:
We will replace all .wsd files with .lutaml files.

Reply:
Ok, but while we didn't accomplish this, why not keep this target? Once we will have converted all wsd lutaml this target will be just ignored